digitalpalireader/content/js/web/chrome_sidebar.js

```diff
@@ -98,7 +98,8 @@ var DPRChrome = {
     var location = document.location.href;
```

Comment:
not related to your changes. since this function is specific to web, can it be moved to dpr_pal?